internal/wire/frame_type_test.go

```diff
@@ -0,0 +1,60 @@
+package wire
+
+import (
+	"testing"
+
+	"github.com/quic-go/quic-go/internal/protocol"
+)
+
+func TestIsStreamFrameType(t *testing.T) {
+	for i := 0x08; i <= 0x0f; i++ {
+		if !FrameType(i).IsStreamFrameType() {
+			t.Errorf("FrameType(0x%x).IsStreamFrameType() = false, want true", i)
+		}
+	}
+
+	if FrameType(0x1).IsStreamFrameType() {
+		t.Errorf("FrameType(0x1).IsStreamFrameType() = true, want false")
+	}
+}
+
+func TestIsAllowedAtEncLevel(t *testing.T) {
```

Comment:
Don't test private methods. They're an implementation details. This should be tested by giving the frame parser various frames to parse, and observing what the public API returns.